.travis.yml

```diff
@@ -57,3 +62,13 @@ env:
  - TEST_SUITE="unittest target=atmega"
 
 script: "scons $TEST_SUITE"
+
+jobs:
+  include:
+    - stage: deploy
+      script:
+        - "python2 tools/authors.py --handles --count --shoutout --since 2017-01-01"
+        - "python3 tools/authors.py --handles --count --shoutout --since 2017-01-01"
+        - "python2 tools/system_design/builder/system_layout.py examples/communication/xml/communication.xml -o /tmp"
+        - "python3 tools/system_design/builder/system_layout.py examples/communication/xml/communication.xml -o /tmp"
```

Comment:
Is this supposed to be duplicated?

Reply:
Yep, as the commit message says, we want to test all scripts with Python 2 and Python 3.

Reply:
Ah, I didn’t see the difference there 🤦‍♂️